devsetup/tripleo/net_config.j2

```diff
@@ -26,37 +26,37 @@ network_config:
     mtu: {{ interface_mtu }}
     vlan_id: 44
     addresses:
-    - ip_netmask: 172.21.0.{{ ip_address_suffix }}/24
+    - ip_netmask: 172.21.0.{{ ip_address_suffix }}/16
```

Comment:
as previously discussed can you please avoid making changes here that affect the 3control/3compute multinode. This file is used for the overcloud node config in that job.

Reply:
(actually this is used for the undercloud node, not overcloud, but same point applies

Reply:
Could we do this for multi-node as well? This change is not breaking, while allows not duplicating more things around

Reply:
possibly but lets please wait until that is merged we are very close now and startign to merge the series

we can discuss next week?

Reply:
I failled back to a flat subnet, there is yet routing issues for /16 external network to address for that switch in ci framework